.github/settings.yml

```diff
@@ -0,0 +1,95 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: breadtubetv
+  # A short description of the repository that will show up on GitHub
+  description: Quality content going against the prevailing winds of the internet.
+  # A URL with more information about the repository
+  homepage: https://breadtube.tv
+  # Either `true` to make the repository private, or `false` to make it public.
+  private: false
+  # Either `true` to enable issues for this repository, `false` to disable them.
+  has_issues: true
+  # Either `true` to enable the wiki for this repository, `false` to disable it.
+  has_wiki: true
+  # Either `true` to enable downloads for this repository, `false` to disable them.
+  has_downloads: true
+  # Updates the default branch for this repository.
+  default_branch: master
+  # Either `true` to allow squash-merging pull requests, or `false` to prevent
+  # squash-merging.
+  allow_squash_merge: true
+  # Either `true` to allow merging pull requests with a merge commit, or `false`
+  # to prevent merging pull requests with merge commits.
+  allow_merge_commit: true
+  # Either `true` to allow rebase-merging pull requests, or `false` to prevent
+  # rebase-merging.
+  allow_rebase_merge: true
+
+# TODO Labels: define labels for Issues and Pull Requests
+labels:
```

Comment:
Does not including this mean that the bot will remove the existing labels?

Reply:
Good call, dunno, removed the key and left the todo. It falls back to existing settings if not defined.